ir/node.cpp

```diff
@@ -94,4 +102,24 @@ Util::JsonObject* IR::Node::sourceInfoJsonObj() const {
     return json;
 }
 
+void IR::Node::sourceInfoToJSON(JSONGenerator &json) const {
+    Util::SourceInfo si = srcInfo;
+    unsigned lineNumber, columnNumber;
+    cstring fName = prepareSourceInfoForJSON(si, &lineNumber, &columnNumber);
```

Comment:
I wondered about this adjustment. I refactored it out of the `sourceInfoJsonObj` code. If I switch to using `toSourcePositionData` in `sourceInfoToJSON`, what should I do in `sourceInfoJsonObj`? I thought it makes sense for both of them to be consistent.